testsuite/MDAnalysisTests/test_coordinates.py

```diff
@@ -1720,6 +1720,22 @@ def test_slice_dcd(self):
         frames = [ts.frame for ts in self.dcd[5:17:3]]
         assert_equal(frames, [5, 8, 11, 14], "slicing dcd [5:17:3]")
 
+    def test_list_trajectory(self):
+        frames = [ts.frame for ts in self.dcd[[0, 3, 4, 5]]]
+        assert_equal(frames, [0, 3, 4, 5])
+
+    def test_array_trajectory(self):
+        frames = [ts.frame for ts in self.dcd[np.array([0, 3, 4, 5])]]
+        assert_equal(frames, [0, 3, 4, 5])
+
+    def test_list_reverse_trajectory(self):
+        frames = [ts.frame for ts in self.dcd[[0, 4, 2, 3, 0, 1]]]
+        assert_equal(frames, [0, 4, 2, 3, 0, 1])
+
+    def test_list_reverse_trajectory(self):
+        frames = [ts.frame for ts in self.dcd[[0, 0, 1, 1, 2, 1, 1]]]
+        assert_equal(frames, [0, 0, 1, 1, 2, 1, 1])
+
     def test_reverse_dcd(self):
```

Comment:
Are the tests in `test_coordinates.py` redundant, given that practically the same tests exist in `test_reader_api.py`? I'm not sure I understand the reason for having both.